gnc/pmc/CMakeLists.txt

```diff
@@ -15,30 +15,26 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.5)
 project(pmc)
 
-## Compile as C++14, supported in ROS Kinetic and newer
-add_compile_options(-std=c++14)
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+set( CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -Wall -O3 -fPIC" )
 
-## Find catkin macros and libraries
-find_package(catkin2 REQUIRED COMPONENTS
-  roscpp
-  config_reader
-)
+# Build for ROS2
+add_definitions(-DROS2)
```

Comment:
I thought we said we wouldn't do this, we would just define it once in the ff_ros.h header for our branch?

Reply:
ignore this for now, I will make a separate pull request